code/__DEFINES/xeno.dm

```diff
@@ -175,7 +175,7 @@
 #define XENO_COMM_ACQUISITION_TIME (55 MINUTES)
 
 /// The time when xenos can grow a destroyer
-#define XENO_KING_ACQUISITION_TIME (1 HOURS + 50 MINUTES)
+#define XENO_KING_ACQUISITION_TIME (1 HOURS + 30 MINUTES + rand(0, 25) MINUTES)
```

Comment:
This define needs to be a constant value because it's used in multiple places and as is each place would now have a different number. If you want randomness you must instead convert this to a global variables instead and use that glob instead of the define.